easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -766,6 +766,14 @@ def remove_false_versions(deps):
                 builddeps = [self._parse_dependency(dep, build_only=True) for dep in builddeps]
             self['builddependencies'] = remove_false_versions(builddeps)
 
+            sourcedeps = self['source_deps']
+            if sourcedeps and all(isinstance(x, (list, tuple)) for b in sourcedeps for x in b):
+                self.iterate_options.append('source_deps')
```

Comment:
`source_deps` should not be an iterative parameter (see also https://docs.easybuild.io/writing-easyconfig-files/#configure_build_install_command_options_iterate)
```suggestion
```